app/src/main/java/org/simple/clinic/drugs/selection/EditMedicinesScreen.kt

```diff
@@ -158,6 +158,6 @@ class EditMedicinesScreen(context: Context, attrs: AttributeSet) : LinearLayout(
   }
 
   override fun showUpdateCustomPrescriptionSheet(patientUuid: UUID, prescribedDrug: PrescribedDrug) {
-    activity.startActivity(CustomPrescriptionEntrySheet.intentForUpdatingPrescription(context, prescribedDrug.uuid))
+    activity.startActivity(CustomPrescriptionEntrySheet.intentForUpdatingPrescription(context, patientUuid, prescribedDrug.uuid))
```

Comment:
`PrescribedDrug` already has a `patientUuid` property, which will be the same. Do we need to pass in a separate `patientUuid` as well or can we get it from the `PrescribedDrug`?

Reply:
Oh right, I missed it 😅 . Will update it.

Reply:
Done